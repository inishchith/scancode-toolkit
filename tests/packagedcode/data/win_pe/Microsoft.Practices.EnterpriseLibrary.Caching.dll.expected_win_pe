{
  "APIVersion": "", 
  "Assembly Version": "2.0.0.0", 
  "BuildDate": "", 
  "BuildVersion": "", 
  "Comments": "Enterprise Library Caching Application Block", 
  "Company": "", 
  "CompanyName": "Microsoft Corporation", 
  "FileDescription": "Enterprise Library Caching Application Block", 
  "FileVersion": "2.0.0.0", 
  "Full Version": "", 
  "InternalName": "Microsoft.Practices.EnterpriseLibrary.Caching.dll", 
  "LegalCopyright": "", 
  "LegalTrademarks": "", 
  "LegalTrademarks1": "", 
  "LegalTrademarks2": "", 
  "License": "", 
  "OriginalFilename": "Microsoft.Practices.EnterpriseLibrary.Caching.dll", 
  "ProductName": "Microsoft Enterprise Library for .NET", 
  "ProductVersion": "2.0.0.0", 
  "WWW": "", 
  "extra_data": {}
}