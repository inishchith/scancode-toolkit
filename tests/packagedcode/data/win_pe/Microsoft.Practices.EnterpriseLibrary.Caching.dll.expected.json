{
  "APIVersion": null, 
  "Assembly Version": "2.0.0.0", 
  "BuildDate": null, 
  "BuildVersion": null, 
  "Comments": "Enterprise Library Caching Application Block", 
  "Company": null, 
  "CompanyName": "Microsoft Corporation", 
  "FileDescription": "Enterprise Library Caching Application Block", 
  "FileVersion": "2.0.0.0", 
  "Full Version": null, 
  "InternalName": "Microsoft.Practices.EnterpriseLibrary.Caching.dll", 
  "LegalCopyright": "", 
  "LegalTrademarks": null, 
  "LegalTrademarks1": null, 
  "LegalTrademarks2": null, 
  "License": null, 
  "OriginalFilename": "Microsoft.Practices.EnterpriseLibrary.Caching.dll", 
  "ProductName": "Microsoft Enterprise Library for .NET", 
  "ProductVersion": "2.0.0.0", 
  "WWW": null, 
  "extra_data": {}
}